docs/pipeline-components-and-applications/version-compatibility-matrix/latest-version-compatibility-matrix/index.md

```diff
@@ -4,50 +4,156 @@ date: "2021-04-29"
+[Enrich](/docs/pipeline-components-and-applications/enrichment-components/index.md) | ${versions.enrich}
+[RDB Loader (Redshift, Snowflake, Databricks)](/docs/destinations/warehouses-and-lakes/rdb/index.md) | ${versions.rdbLoader}
+[S3 Loader](/docs/destinations/warehouses-and-lakes/s3/index.md) | ${versions.s3Loader}
+[Elasticsearch Loader](/docs/destinations/forwarding-events/elasticsearch/index.md) | ${versions.esLoader}
+[Postgres Loader](/docs/destinations/warehouses-and-lakes/postgres/index.md) | ${versions.postgresLoader}
+[sqs2kinesis](/docs/pipeline-components-and-applications/sqs2kinesis/index.md) | ${versions.sqs2kinesis}
+[Dataflow Runner](/docs/pipeline-components-and-applications/dataflow-runner/index.md) | ${versions.dataflowRunner}
+`} remarkPlugins={[remarkGfm]} />
+
+</TabItem>
+<TabItem value="gcp" label="GCP">
 
-Please note, the `x` indicates that we recommend always being on the latest patched version.
+<ReactMarkdown children={`
+Component | Latest version
+:--|:-:
+[Stream Collector](/docs/pipeline-components-and-applications/stream-collector/index.md) | ${versions.collector}
+[Enrich](/docs/pipeline-components-and-applications/enrichment-components/index.md) | ${versions.enrich}
+[RDB Loader (Snowflake, Databricks)](/docs/destinations/warehouses-and-lakes/rdb/index.md) | ${versions.rdbLoader}
+[BigQuery Loader](/docs/destinations/warehouses-and-lakes/bigquery/index.md) | ${versions.bqLoader}
+[GCS Loader](/docs/destinations/warehouses-and-lakes/google-cloud-storage/index.md) | ${versions.gcsLoader}
+[Postgres Loader](/docs/destinations/warehouses-and-lakes/postgres/index.md) | ${versions.postgresLoader}
+`} remarkPlugins={[remarkGfm]} />
 
-#### AWS
+</TabItem>
+</Tabs>
+
+## Iglu (schema registry)
+
+:::info Snowplow BDP
+
+If you are using Snowplow BDP (Enterprise or Cloud), you don’t need to install Iglu Server yourself. It’s also unlikely that you need to use any of the other components in this section. You can manage your data structures [in the UI](/docs/understanding-tracking-design/managing-data-structures/index.md) or [via the API](/docs/understanding-tracking-design/managing-data-structures-via-the-api-2/index.md).
+
+:::
+
+<ReactMarkdown children={`
+Component | Latest version
+:--|:-:
+[Iglu Server](/docs/pipeline-components-and-applications/iglu/iglu-repositories/iglu-server/index.md) | ${versions.igluServer}
+[\`igluctl\` utility](/docs/pipeline-components-and-applications/iglu/igluctl-2/index.md) | ${versions.igluctl}
+[Iglu Scala client](/docs/pipeline-components-and-applications/iglu/iglu-clients/scala-client-setup/index.md) | ${versions.igluScalaClient}
+[Iglu Ruby client](/docs/pipeline-components-and-applications/iglu/iglu-clients/ruby-client/index.md) | ${versions.igluRubyClient}
+[Iglu Objective-C client](/docs/pipeline-components-and-applications/iglu/iglu-clients/objc-client/index.md) | ${versions.igluObjCClient}
+`} remarkPlugins={[remarkGfm]} />
+
+## Trackers
+
+<ReactMarkdown children={`
```

Comment:
@mscwilson FYI — I got the versions from GitHub releases. Could you check if any of these should not be in the list because they are deprecated? (And I will need your help to move these versions to variables, but that’s for another PR.)